book/history/sourdough-history.tex

```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
     lessons from the past.
 \end{quoting}
 
-Sourdough has been made since ancient times. The exact origins of fermented
+The story of sourdough bread begins in ancient oceans. These oceans were the
+birthplace of all Earth's life. To understand this timeline, envision Earth's
+history compressed into one year. On this scale, January 1st signifies Earth's
+formation 4.54 billion years ago. Midnight on December 31st is our present.
+Humans appeared on the evening of December 31st. The dinosaurs, once our
+planet's rulers, vanished on December 29th. Earlier on that same day, humans
+made their debut. Just seconds ago, the first firecrackers were lit.
+
+On March 25th, the oceans birthed the first single-celled bacteria. In these
+waters, another single-celled life form, \emph{archaea}, also thrived. These
+organisms inhabit extreme environments, from boiling vents to icy waters.
+
+\begin{figure}[!htb]
+\begin{center}
+  \input{figures/fig-life-planet-sourdough-timeline.tex}
+  \caption[Sourdough microbiology timeline]{Timeline of significant events
+    starting from the first day of Earth's existence,
+    divided into months, and extending to the present day,
+    marked at midnight. This visualization shows the pivotal steps
+    of life and sourdough on earth.}%
+\end{center}
+\end{figure}
```

Comment:
Not sure this is clearer than showing millions of years… I see why you are trying to say but I need to think about it.

Reply:
I got the idea from Ed Young's book "I contain multitudes". I kind of like it as it shows how recent all human innovation has been. Also shows how lonely the planet has been for some time. My plan is to add another one with 12000 BC till now once this PR is merged 😅

Reply:
Have you tried to also have non scaled dates maybe in a different colour?

I would also add some key dates like Pangea (you already have dinosaurs extinction).  Just an idea.

Reply:
Pangea is a funny idea 👍 . Just not sure I can fit more into that one chart 😅

Reply:
Also - TLDR - it was that recent the continents formed 😅